GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2717,8 +2717,8 @@ private void editFileToolStripMenuItem_Click(object sender, EventArgs e)
                 return;
             }
 
-            UICommands.StartFileEditorDialog(fileName);
-
+            int? lineNumber = SelectedDiff.CurrentFileLine;
+            UICommands.StartFileEditorDialog(fileName, lineNumber: lineNumber);
```

Comment:
❔ no need for a variable in this case
```suggestion
            UICommands.StartFileEditorDialog(fileName, lineNumber: SelectedDiff.CurrentFileLine);
```